catalog-rest-service/src/main/resources/ui/src/components/common/entityPageInfo/EntityPageInfo.tsx

```diff
@@ -26,26 +26,29 @@ type ExtraInfo = {
 
 type Props = {
   titleLinks: TitleBreadcrumbProps['titleLinks'];
-  isFollowing: boolean;
-  followHandler: () => void;
-  followers: number;
+  isFollowing?: boolean;
+  followers?: number;
   extraInfo: Array<ExtraInfo>;
   tier: string;
   tags: Array<EntityTags>;
   isTagEditable?: boolean;
   tagList?: Array<string>;
   owner?: TableDetail['owner'];
   hasEditAccess?: boolean;
-  tagsHandler?: (selectedTags?: Array<string>) => void;
   followersList: Array<User>;
   entityName: string;
+  version?: string;
+  isVersionSelected?: boolean;
+  followHandler?: () => void;
+  tagsHandler?: (selectedTags?: Array<string>) => void;
+  versionHandler?: () => void;
 };
 
 const EntityPageInfo = ({
   titleLinks,
   isFollowing,
   followHandler,
-  followers,
+  followers = 0,
```

Comment:
Why `0` by default?

Reply:
if there are no followers then by default we should show it to zero.